src/components/organisms/Vault/vault.organism.types.ts

```diff
@@ -12,3 +12,12 @@ export interface IVaultProps {
   account: Nullable<string>;
   provider: JsonRpcProvider;
 }
+
+export interface IOnChangeHandler {
```

Comment:
There's an onChange handler type in input.atom.types, we can move it to util.types.ts and use that one